wire-ios/Wire-iOS/Sources/UserInterface/AddContacts/AddParticipantsViewController.swift

```diff
@@ -581,3 +585,13 @@ extension AddParticipantsViewController: EmptySearchResultsViewDelegate {
         }
     }
 }
+
+extension Int {
+    var participantsFormatted: String {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .decimal
+        formatter.locale = Locale.current
+
+        return formatter.string(from: NSNumber(value: self)) ?? String(self)
+    }
+}
```

Comment:
issue: please don't create unrelated extensions to existing types, it spoils the namespace.

<img width="704" alt="Screenshot 2025-05-23 at 09 29 00" src="https://github.com/user-attachments/assets/b5c1f57a-cf85-4c26-ac79-b01e1812635d" />